rhel7/fixes/bash/sshd_use_approved_ciphers.sh

```diff
@@ -3,4 +3,4 @@
 # Include source function library.
 . /usr/share/scap-security-guide/remediation_functions
 
-replace_or_append '/etc/ssh/sshd_config' '^Ciphers' 'aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc' '@CCENUM@' '%s %s'
```

Comment:
Can you make this a variable? DISA feels only a subset of FIPS cipbers should be used; however, other gov agencies will use any FIPS cipher.

Reply:
Both the OVAL, XCCDF, and profile will have to be updated for the variable. If you need help on that, please let us know.